tests/regression/c_bindings/implementation.h

```diff
@@ -1,12 +1,3 @@
-/*
- * GridTools
- *
- * Copyright (c) 2014-2021, ETH Zurich
- * All rights reserved.
- *
- * Please, refer to the LICENSE file in the root directory.
- * SPDX-License-Identifier: BSD-3-Clause
- */
```

Comment:
Just wondering about these changes, seem to be totally unrelated, but don’t really care.

Reply:
Yes, these generated files got accidentally the license header. Just sneaked it in here.